src/java/htsjdk/samtools/cram/build/ContainerParser.java

```diff
@@ -144,14 +204,15 @@ public ContainerParser(final SAMFileHeader samFileHeader) {
             if (!nanosecondsMap.containsKey(key)) {
                 nanosecondsMap.put(key, 0L);
                 value = 0;
-            } else
+            } else {
                 value = nanosecondsMap.get(key);
+            }
             nanosecondsMap.put(key, value + statMap.get(key).nanos);
         }
```

Comment:
Is this code only used for perf analysis/instrumentation ? Can we remove it - it looks like some of the supporting code is exposed through the public API (i.e, Container.parseTime, buildHeaderTime, etc) ?